### PR TITLE
Enable passing custom domain username via cli

### PR DESCRIPTION
**Pull request by chethanuk**

### CODE REVIEW

File: backend/pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [project]
 name = "tidb-ai-python"
-version = "0.1.0"
-description = "Add your description here"
+version = "0.2.9"
+description = "TIDB AI Python Backend"
```
Comment (sykp241095) [inline backend/pyproject.toml:4]:
```suggestion
description = "AutoFlow Python Backend"
```

File: backend/bootstrap.py [hunk 2]
```diff
@@ -3,19 +3,20 @@
 from sqlmodel import select, func
 from sqlmodel.ext.asyncio.session import AsyncSession
 from colorama import Fore, Style
+import click
```
Comment (sykp241095) [inline backend/bootstrap.py:6]: Hi, we should add it to pyproject.toml via poetry?

Comment (chethanuk) [reply backend/bootstrap.py:6]: Yes

Comment (sykp241095) [reply backend/bootstrap.py:6]: @chethanuk Yes, I mean could you help submit click to pyproject 😂

**Review by sykp241095 — APPROVED** [review]

**Pull request merged**